### PR TITLE
Add Flang include directory to MSVC toolchain

### PR DESCRIPTION
**Pull request by kaadam**

### CODE REVIEW

**Comment by kaadam** [comment]
@bryanpkc @pawosm-arm Hi, Could you take a look at this PR please? Thanks!

File: clang/lib/Driver/ToolChains/MSVC.cpp [hunk 1]
```diff
@@ -689,6 +689,42 @@ void MSVCToolChain::AddSystemIncludeWithSubfolder(
...
+                                  ToolChain::path_list IncludePathList) {
+  std::string ArgValue; // Path argument value
+
+  // Make up argument value consisting of paths separated by colons
+  bool first = true;
+  for (auto P : IncludePathList) {
+    if (first) {
+      first = false;
+    } else {
+      ArgValue += ":";
```
Comment (pawosm-arm) [inline clang/lib/Driver/ToolChains/MSVC.cpp]: using `:` as a separator in the Windows world seems tricky, how would it look like? `"c:\msvc:d:\projects"`, is this approach ok?

Comment (kaadam) [reply clang/lib/Driver/ToolChains/MSVC.cpp]: No, officially the separator is the semicolon ';' on Windows. Thanks for the review. Switching between the terminals, (mingw, ms command prompt) confuses me sometimes. But ':' is also works. I will update the separator.

**Comment by kaadam** [comment]
@bryanpkc, @pawosm-arm I've updated the PR, could you look at it again, please?

**Review by pawosm-arm — APPROVED** [review]

**Review by bryanpkc — APPROVED** [review]

**Pull request merged**